test/redpanda/mzworkflows.py

```diff
@@ -32,5 +31,5 @@ def workflow_redpanda_testdrive(w: Workflow):
 
     w.run_service(
         service="testdrive-svc",
-        command="grep -L -E 'kafka_time_offset|schema-type=protobuf' testdrive/*.td",
+        command="grep -L -E 'kafka_time_offset' testdrive/*.td",
```

Comment:
can't wait to squash this last bug, and then would be dope to move to `:latest` tag once the last one is there so we continue to have perfect compat.